diff -up kparts-5.72.0/src/part.h.omv~ kparts-5.72.0/src/part.h
--- kparts-5.72.0/src/part.h.omv~	2020-07-18 00:38:32.840317816 +0200
+++ kparts-5.72.0/src/part.h	2020-07-18 00:38:51.031521536 +0200
@@ -38,7 +38,7 @@ class PartManager;
 class PartPrivate;
 class PartActivateEvent;
 
-#if KPARTS_ENABLE_DEPRECATED_SINCE(5, 72)
+#if KPARTS_BUILD_DEPRECATED_SINCE(5, 72)
 class PartSelectEvent;
 #endif
 
